app/services/migration/migrators/application.rb

```diff
@@ -71,6 +71,13 @@ def call
 
         if ecf_npq_application.school_urn.present?
           application.school_id = find_school_id!(urn: ecf_npq_application.school_urn)
+
+          if application.school.ukprn.to_s == ecf_npq_application.school_ukprn.to_s
```

Comment:
Would this check not make more sense in the schools migrator? otherwise we'll end up with multiple errors for the same school UKPRN mismatch (one for every application at the school). Whereas if we did it in the schools migrator it would fail once for the school it was wrong for? something like:

```
migrate(self.class.ecf_schools) do |ecf_school|
  school = find_school_id!(urn: ecf_school.urn, name: ecf_school.name.downcase)
  raise if school.ukprn != ecf_school.ukprn
end
```

Reply:
This is testing if `ecf_npq_application.school_urn` and `ecf_npq_application.school_ukprn` both point to the same school. Therefore the application record has a mismatch.